src/test/cli/monmaptool/feature-set-unset-list.t

```diff
@@ -11,64 +11,64 @@
 
   $ monmaptool --feature-set foo /tmp/test.monmap.1234
   unknown features name 'foo' or unable to parse value: Expected option value to be integer, got 'foo'
   monmaptool -h for usage
   [1]
 
-  $ monmaptool --feature-set kraken --feature-set 64 --optional --feature-set 32 --persistent /tmp/test.monmap.1234
+  $ monmaptool --feature-set kraken --feature-set 4096 --optional --feature-set 32 --persistent /tmp/test.monmap.1234
   monmaptool: monmap file /tmp/test.monmap.1234
   monmaptool: writing epoch 0 to /tmp/test.monmap.1234 (1 monitors)
 
   $ monmaptool --feature-list /tmp/test.monmap.1234
   monmaptool: monmap file /tmp/test.monmap.1234
   MONMAP FEATURES:
-      persistent: [kraken(1),unknown(32)]
-      optional:   [unknown(64)]
-      required:   [kraken(1),unknown(32),unknown(64)]
+      persistent: [kraken(1),octopus(32)]
+      optional:   [unknown(4096)]
+      required:   [kraken(1),octopus(32),unknown(4096)]
   
   AVAILABLE FEATURES:
-      supported:  [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16)]
-      persistent: [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16)]
+      supported:  [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16),octopus(32)]
+      persistent: [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16),octopus(32)]
 
   $ monmaptool --feature-unset 32 --optional --feature-list /tmp/test.monmap.1234
   monmaptool: monmap file /tmp/test.monmap.1234
   MONMAP FEATURES:
-      persistent: [kraken(1),unknown(32)]
-      optional:   [unknown(64)]
-      required:   [kraken(1),unknown(32),unknown(64)]
+      persistent: [kraken(1),octopus(32)]
+      optional:   [unknown(4096)]
+      required:   [kraken(1),octopus(32),unknown(4096)]
   
   AVAILABLE FEATURES:
-      supported:  [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16)]
-      persistent: [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16)]
+      supported:  [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16),octopus(32)]
+      persistent: [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16),octopus(32)]
   monmaptool: writing epoch 0 to /tmp/test.monmap.1234 (1 monitors)
 
-  $ monmaptool --feature-unset 32 --persistent --feature-unset 64 --optional --feature-list /tmp/test.monmap.1234
+  $ monmaptool --feature-unset 32 --persistent --feature-unset 4096 --optional --feature-list /tmp/test.monmap.1234
   monmaptool: monmap file /tmp/test.monmap.1234
   MONMAP FEATURES:
       persistent: [kraken(1)]
       optional:   [none]
       required:   [kraken(1)]
   
   AVAILABLE FEATURES:
-      supported:  [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16)]
-      persistent: [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16)]
+      supported:  [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16),octopus(32)]
+      persistent: [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16),octopus(32)]
```

Comment:
might want to update `TEST_mon_features()` in `qa/standalone/mon/misc.sh`. 

for instance.
1. check that we have "octopus" in monitor features
1. `.monmap.features.persistent | length == *6*`

Reply:
fixed!